plotly/tools.py

```diff
@@ -2275,6 +2277,25 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None, **kwargs):
+        """
```

Comment:
@cldougl seems like I can't access this link, but I tried to comply anyways =) have a look at 57e15bdffbd5f4ffd00d0e349e8346116ab4ee56